definitions/mobile.d.ts

```diff
@@ -314,7 +314,7 @@ declare module Mobile {
 	}
 
 	interface IDeviceDiscovery extends NodeJS.EventEmitter {
-		startLookingForDevices(platform?: string): Promise<void>;
+		startLookingForDevices(platform?: string, options?: IDeviceLookingOptions): Promise<void>;
```

Comment:
lets add the `platform` in  `Mobile.IDeviceLookingOptions` - this will make the code cleaner. We've added it for external device discoveries, but currently noone respects the platform